src/get-started/design-overview.md

```diff
@@ -118,7 +118,7 @@ Full nodes are further subdivided into 2 categories:
   [Byzantine fault tolerant consensus algorithm](#consensus).
   Validators receive transactions, verify them, and include into a new block.
   The list of the validators is restricted by network maintainers, and normally
-  should consist of 4–15 nodes
+  should consist of 4–16 nodes
```

Comment:
This is not actually a typo, but it seems to me that for `>2/3` consensus, it doesn't make sense to run `15` nodes instead of `13`? Besides, now both 4 and 16 are round numbers :)

Reply:
The restriction of 15 nodes is related to Bitcoin anchoring. 15 is the maximum number of validators that allows to create anchoring transactions considered [standard](https://bitcoin.org/en/developer-guide#standard-transactions). (Although this should've changed with SegWit, the anchoring service does not have SegWit support yet.)